<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MappingWidget</class>
 <widget class="QWidget" name="MappingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MappingWidget { 
background-color: rgb(229, 243, 255);
}
</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>264</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QToolButton" name="exportButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Export as .csv</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{background-color:transparent};</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/Resources/export.png</normaloff>:/Resources/export.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="styleSheet">
       <string notr="true">QTreeView::item:hover {backgroundcolor:rgb(229,243,255);
														selection-color: black;}
QTreeView::item:selected {background-color: rgb(21, 153, 255);
																selection-color: white;}
QHeaderView::section {background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                			    stop: 0 			#7aacd7, stop: 0.614 white,
                                				 stop: 0.614  #FF007F96,  stop: 0.853 white
													 stop: 0.853 	#7aacd7, stop: 1.0 white);
													 color: rgb(255,255,255);
                                              border-width: 1px; border-style: solid;
													 border-color: #D8D8D8;
                                              height: 40px; padding-left:10px;
													 font: 10pt \Open Sans\;}</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>228</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="importButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>18</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
